src/utils/Constants.ts

```diff
@@ -83,10 +83,8 @@ export default class Constants {
   public static readonly WS_RECONNECT_UNLIMITED = -1;
   public static readonly WS_DEFAULT_RECONNECT_MAX_RETRIES = -1;
   public static readonly WS_DEFAULT_RECONNECT_TIMEOUT = 30; // Seconds
-  public static readonly WS_UNSUPPORTED_DATA = 1007;
 
-  public static readonly OCPP_SOCKET_TIMEOUT = 30000; // 30 sec
-  public static readonly OCPP_RESPONSE_ACCEPTED = 'Accepted';
+  public static readonly OCPP_ERROR_TIMEOUT = 30000; // 30 sec
```

Comment:
Error Timeout!?
OCPP_SOCKET_TIMEOUT was more meaningful

Reply:
> Error Timeout!?
> OCPP_SOCKET_TIMEOUT was more meaningful

It's a timeout used at error OCPP message sending. It has nothing to do with a WS timeout: the callback is using a responsive WS connection (it's a requirement to send an OCPP message) and it timeout because the charging station does not respond to the error message.

Reply:
You were right. I will revert that change.